packages/forma-36-react-components/src/components/RadioButton/RadioButton.stories.tsx

```diff
@@ -1,34 +1,38 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { text, boolean } from '@storybook/addon-knobs';
 
-import RadioButton from './RadioButton';
+import RadioButton, { RadioButtonProps } from './RadioButton';
 import Flex from '../Flex/Flex';
 import SectionHeading from '../Typography/SectionHeading';
 
-storiesOf('Components/RadioButton', module)
-  .addParameters({
-    propTypes: RadioButton['__docgenInfo'],
-    component: RadioButton,
-  })
-  .add('default', () => (
-    <RadioButton
-      className={text('className', '')}
-      id="Checkbox"
-      checked={boolean('Checked', false)}
-      labelText={(text('Aria label text'), 'some label text')}
-      disabled={boolean('Disabled', false)}
-      required={boolean('Required', false)}
-      name={text('Name', 'some-name')}
-      willBlurOnEsc={boolean('willBlurOnEsc', true)}
-    />
-  ))
-  .add('overview', () => (
-    <>
+export default {
+  title: 'Components/RadioButton',
+  component: RadioButton,
+  parameters: {
+    propTypes: [RadioButton['__docgenInfo']],
+  },
+  argTypes: {
+    className: { control: { disable: true } },
+    testId: { control: { disable: true } },
+  },
+};
+
+export const basic = (args: RadioButtonProps) => <RadioButton {...args} />;
+
+basic.args = {
+  id: 'Checkbox',
+  labelText: 'some label text',
```

Comment:
This does not look like it's doing anything in the story, is it really a prop in RadioButton?